addons/hr_holidays/static/src/scss/time_off.scss

```diff
@@ -41,4 +41,18 @@
     .o_timeoff_green {
         color: $o-enterprise-primary-color;
     }
+
+    .o_calendar_view .fc-dayGridYear-view {
+        padding-top: 0 !important;
```

Comment:
I would at least keep this padding here

Reply:
No, I removed all vertical spaces in time_off because the height of fc-dayGridYear-view is bigger than its container
so the scroller appears and we don't want to have this scroller.
Ok it doesn't change anything to keep the padding for this year but look at 2021 it will show the scroller

Reply:
Which scrollbar are you talking about? 'cause on my laptop screen, I have a scrollbar no matter the padding values or the displayed year...